### PR TITLE
Pico VS Code extension

### PR DESCRIPTION
**Pull request by sungshik**
This PR extends the syntax-highlighting-only VS Code extension (with generated TextMate grammars) with support for Pico (in addition to Rascal). To this end, this PR:

  - Makes a few changes to the configuration of the VS Code extension
  - Reorganizes the code to generate grammars for use in the VS Code extension

Note: None of the actual conversion code is affected.

### CODE REVIEW

File: rascal-textmate-core/src/main/rascal/VSCodeRascal.rsc [hunk 1]
```diff
@@ -0,0 +1,61 @@
+@synopsis{
+    Main function to generate a new TextMate grammar for Rascal, to be used in
+    the special VS Code extension
+}
+
+module VSCodeRascal
```
Comment (sungshik) [inline rascal-textmate-core/src/main/rascal/VSCodeRascal.rsc:6]: This is essentially a rename of the old `VSCode` module:

  - Function `main` was a bit streamlined
  - Function `getRscGrammar` is identical to before

File: rascal-textmate-core/src/main/rascal/lang/textmate/conversiontests/PicoWithCategories.rsc [hunk 2]
```diff
@@ -42,10 +42,22 @@ syntax Expression
            | min: Expression lhs "-" Expression rhs)
     ;
            
-lexical Id = [a-z][a-z0-9]* !>> [a-z0-9];
+lexical Id = ([a-z][a-z0-9]*) !>> [a-z0-9] \ Keyword;
```
Comment (sungshik) [inline rascal-textmate-core/src/main/rascal/lang/textmate/conversiontests/PicoWithCategories.rsc:45]: Added this condition (diverging slightly from `lang::pico::\syntax::Main`) to make the highlighting a bit nicer)

**Review by sungshik — COMMENTED** [review]
A few review comments

**Review by DavyLandman — APPROVED** [review]
LGTM 👍🏼

**Pull request merged**